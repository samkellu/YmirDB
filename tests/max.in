max a
set a 1 2 3 4 5 6 22 34
set b 123 334 a 44
set c b a 12 55 7
max c
bye
