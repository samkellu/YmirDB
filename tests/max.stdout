==2853== Memcheck, a memory error detector
==2853== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==2853== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==2853== Command: ./ymirdb
==2853== 
==2853== error calling PR_SET_PTRACER, vgdb might block
> No such entry

> ok

> ok

> no such key

> No such entry

> bye
==2853== 
==2853== HEAP SUMMARY:
==2853==     in use at exit: 0 bytes in 0 blocks
==2853==   total heap usage: 43 allocs, 43 frees, 2,536 bytes allocated
==2853== 
==2853== All heap blocks were freed -- no leaks are possible
==2853== 
==2853== For lists of detected and suppressed errors, rerun with: -s
==2853== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
